src/db/core.ts

```diff
@@ -383,11 +383,22 @@ export class SQLFragment<RunResult = pg.QueryResult['rows'], Constraint = never>
           columnNames = <Column[]>Object.keys(expression.value).sort(),
```

Comment:
This block could be simplified if the columns are filtered here. Then there is no need for `firstField`

```
           columnNames = <Column[]>Object.keys(expression.value)
             .filter(key => (<any>expression.value)[key] !== undefined)
             .sort(),
```